queue/setup.go

```diff
@@ -37,11 +39,14 @@ type Setup struct {
 func (s *Setup) Redis() (Service, error) {
 	logrus.Trace("creating redis queue client from setup")
 
-	// check if the default route is provided
-	if !strings.Contains(strings.Join(s.Routes, ","), constants.DefaultRoute) {
+	// check if default route is to be used
+	// and append it if it is not included in routes
+	if !s.SkipDefaultRoute && !strings.Contains(strings.Join(s.Routes, ","), constants.DefaultRoute) {
 		s.Routes = append(s.Routes, constants.DefaultRoute)
 	}
 
+	logrus.Debugf("redis queue routes set to: %s", s.Routes)
```

Comment:
Should this be `trace` to match the above trace logs in this package?

Reply:
sure, i can match - thanks for taking a look